test/__snapshots__/main.test.js.snap

```diff
@@ -0,0 +1,41 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Router instance methods getTestFileCallback: insecure 1`] = `
+"<!doctype html>
```

Comment:
if you pass the object through `jsdom` it gets pretty printed. not sure if worth it, though

Reply:
oooh protip 🖖

Reply:
I can't see any way to pretty print using jsdom. Can I get a "pro clue". Woop woop. 😬

Reply:
(internal link) https://github.schibsted.io/finn/podium/blob/master/test-helpers.js

then `toMatchSnapshot` will work as it includes the `pretty-format` plugin for html elements

Reply:
Or I could just use something like: https://www.npmjs.com/package/pretty

Reply:
Sure, but we already have pretty-format (and jsdom) through jest, and it already supports pretty-printing html